tests/cxx/include_with_using.cc

```diff
@@ -41,7 +41,6 @@ void TestDeclFromD4() {
   var_in_d4 = 0;
 }
 
-// IWYU: i1::UsingInCc needs a declaration
```

Comment:
Cool, I understand now that it's the deferred analysis that does it, and I ran it again locally and saw that the warning actually happens on line 48 instead. Now I'm happy! :-)